src/pages/home/report/ReportActionItemParentAction.tsx

```diff
@@ -81,6 +83,19 @@ function ReportActionItemParentAction({
     const [allAncestors, setAllAncestors] = useState<Ancestor[]>([]);
     const {isOffline} = useNetwork();
     const {isInNarrowPaneModal} = useResponsiveLayout();
+    const [reportNameValuePairs] = useOnyx(ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS, {
+        canBeMissing: true,
+        selector: (allPairs) => {
+            const ancestorIDsToSelect = new Set(ancestorIDs.current.reportIDs);
```

Comment:
This can be moved outside of the selector so that it doesn't have to run on every iteration.